sensor_msgs/Image image
sensor_msgs/Image mask
string label
---
bool success
